### PR TITLE
Update build instructions

### PR DESCRIPTION
**Pull request by webmaster128**

### CODE REVIEW

File: testnets/build-requirements.md [hunk 1]
```diff
@@ -21,27 +21,30 @@ Use docker image, or build locally: `https://github.com/CosmWasm/wasmd/#dockeriz
...
-# docker cp $id:/usr/bin/coralcli - > coralcli
-docker rm -v $id
-```
+   ```sh
+   id=$(docker create wasmd-docker)
+   docker cp $id:/usr/bin/wasmd - > wasmd
+   docker cp $id:/usr/bin/wasmcli - > wasmcli
+   chmod +x wasmd wasmcli
+   # docker cp $id:/usr/bin/corald - > corald
+   # docker cp $id:/usr/bin/coral - > coral
```
Comment (webmaster128) [inline testnets/build-requirements.md]: Wait a second … docker cp seems to create a .tar file that needs to be extracted. Will update

Comment (webmaster128) [reply testnets/build-requirements.md]: From `docker cp --help`:

> Use '-' as the destination to stream a tar archive of a container source to stdout.

Comment (webmaster128) [reply testnets/build-requirements.md]: Done in https://github.com/CosmWasm/docs2/pull/54/commits/0fe82270dfa26c0a220078b7a3f6054fa892f0de

**Review by orkunkl — APPROVED** [review]
Nice job!

**Pull request merged**